pkg/apply/basic_printer.go

```diff
@@ -132,3 +133,12 @@ func getName(obj runtime.Object) string {
 func resourceIDToString(gk schema.GroupKind, name string) string {
 	return fmt.Sprintf("%s/%s", strings.ToLower(gk.String()), name)
 }
+
+func (b *BasicPrinter) getPrintFunc(preview bool) func(message string) {
+	return func(message string) {
```

Comment:
Could we copy the method signature of `Fprintf` here so we could accept a template and a variable number of args? This way, we don't have to do `Sprintf` every time before calling `printFunc`

Reply:
Done. Thanks